src/vivarium_csu_hypertension_sdc/components/builder.py

```diff
@@ -159,6 +159,11 @@ def write_ckd_data(artifact, location):
 
 def write_sbp_data(artifact, location):
     load = get_load(location)
+    affected_entity_map = {'ischemic_heart_disease': 'acute_myocardial_infarction',
```

Comment:
Only storing used causes and need to rename disease rates to correspond with actual states.